tools/manager/match.py

```diff
@@ -56,63 +57,37 @@ def __repr__(self):
         return title1 + title2 + dims + results + replay
 
     def get_command(self, halite_binary):
-        dims = "-d " + str(self.map_width) + " " + str(self.map_height)
+        if self.keep_replay is True:
+            replays = "--replay-directory replays/"
+        else:
+            replays = "--no-replay"
+
+        if self.keep_logs is True:
+            logs = ""
+        else:
+            logs = "--no-logs"
+        width = "--width 32"
+        height = "--height 32"
         quiet = "-q"
         seed = "-s " + str(self.map_seed)
-        result = [halite_binary, dims, quiet, seed]
+        results = "--results-as-json"
+        result = [halite_binary, replays, logs, width, height, seed, results]
         return result + self.paths
 
     def run_match(self, halite_binary):
         command = self.get_command(halite_binary)
         print("Command = " + str(command))
         p = Popen(command, stdin=None, stdout=PIPE, stderr=None)
         results, _ = p.communicate(None, self.total_time_limit)
+
         self.results_string = results.decode('ascii')
         self.return_code = p.returncode
         self.parse_results_string()
         update_skills(self.players, copy.deepcopy(self.results))
-        if self.keep_replay:
-            print("Keeping replay\n")
-            if not os.path.exists(record_dir):
-                os.makedirs(record_dir)
-            try:
-                shutil.move(self.replay_file, record_dir)
-                if self.replay_file.startswith('./'):
-                    self.replay_file = self.replay_file[2:]
-                self.replay_file = os.path.join(record_dir, self.replay_file)
-            except Exception as e:
-                print(e)
-        else: 
-            print("Deleting replay\n")
-            os.remove(self.replay_file)
-
-        if self.keep_logs:
-            print("Keeping logs\n")
-            if not os.path.exists(record_dir):
-                os.makedirs(record_dir)
-            try:
-                for key, filename in self.logs.items():
-                    shutil.move(filename, record_dir)
-                    self.logs[key] = os.path.join(record_dir, filename)
-            except Exception as e:
-                print(e)
-        else: 
-            print("Deleting logs\n")
-            for file in self.logs.values():
-                os.remove(file)
-#        self.fix_logs()
-
-#    def fix_logs(self):
-#        print("Fixing logs")
-#        for index in range(0, self.num_players):
-#            if not self.logs[index]:
-#                self.logs[index] = None
-#        print(self.logs)
-        
 
     def parse_results_string(self):
         data = json.loads(self.results_string)
-        self.parameters = data['gameplay_parameters']
+        self.final_snapshot = data['final_snapshot']
```

Comment:
You can totally ignore this; it's an unfinished feature.

Reply:
Done.